web/wp-content/themes/dctx-v2/template-sidebar-right.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-3rfsjx.php, but found phpcs-scan-3rfsJX.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).